post-build/action.yml

```diff
@@ -57,7 +57,8 @@ runs:
     - name: Count bottles
       id: bottles
       if: always()
-      run: '"${GITHUB_ACTION_PATH}/count-bottles.sh" "${{ inputs.debug }}"'
+      run: |
+        "${GITHUB_ACTION_PATH}/count-bottles.sh" ${{ runner.debug == '1' }}
```

Comment:
I think there's no injection potential here, but even still it might be nice to put the `${{ .. }}` expansion into an envvar for consistency 🙂

Reply:
Also: double check that this fails as expected on a non-zero exit code; somethings moving to using `|` breaks this.